lxc/completion.go

```diff
@@ -1634,9 +1634,17 @@ func (g *cmdGlobal) cmpStoragePoolVolumes(poolName string, volumeTypes ...string
 
 	customVolumeNames := make([]string, 0, customKeyCount)
 
-	// Only include custom volumes
+	// Only complete volumes specified by volumeTypes.
 	for _, volume := range volumes {
 		_, volType := parseVolume("custom", volume)
+
+		// Parse snapshots returned by GetStoragePoolVolumeNames.
+		if strings.Contains(volume, "/snapshots/") {
+			fields := strings.SplitN(volume, "/snapshots/", 2)
+			customVolumeNames = append(customVolumeNames, fields[0]+"/"+fields[1])
+			continue
+		}
```

Comment:
Feels like a good candidate for `before, after, found := strings.Cut` as it combines the `strings.Contains` and `strings.SplitN` into one operation.